src/main/java/org/openshift/jenkins/plugins/openshiftlogin/OpenShiftOAuth2SecurityRealm.java

```diff
@@ -257,11 +258,13 @@ public void initialize(HttpRequest request) throws IOException {
                                 }
                             });
                     GenericUrl url = new GenericUrl(serverPrefix + "/oapi/v1/users/~");
-
+                    
+                    // do self-sar here to see if you have the "admin" or "view" verb in the "build.openshift.io" api group on the "jenkins" resource 
```

Comment:
So I can't figure out how to get permssion to create the subjectaccessreview ... with the url ending in `oapi/v1/subjectaccessreviews` I get a message saying I can't create those at the cluster scope.  I tried inserting a `namespaces/<my namespace>` in the appropriate spot, and it says I can't create a subjectaccessreview in namespace specified by `<my namspace>`

What roles do I need to add to the user I'm logging in as?